src/main/java/org/profit/telegrambot/controller/MainController.java

```diff
@@ -0,0 +1,122 @@
+package org.profit.telegrambot.controller;
+
+import org.telegram.telegrambots.meta.api.methods.ParseMode;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.methods.updatingmessages.DeleteMessage;
+import org.telegram.telegrambots.meta.api.objects.Contact;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardRemove;
+import org.profit.telegrambot.container.ComponentContainer;
+import org.profit.telegrambot.enums.CustomerStatus;
+import org.profit.telegrambot.model.Customer;
+import org.profit.telegrambot.service.CustomerService;
+import org.profit.telegrambot.util.InlineKeyboardUtil;
+import org.profit.telegrambot.util.KeyboardUtil;
+
+public class MainController {
+
+    public void handleMessage(Message message) {
+        if (message.hasText()) {
+            handleText(message);
+        } else if (message.hasContact()) {
+            handleContact(message);
+        }
```

Comment:
Usually, we log something in case of a method does nothing (or at least leaving a comment).

Here, if the message has neither text nor a contact, no if-else branch will be executed. 
You can dig deeper and go over a loggin framework (e.g. log4j), but simple `System.out.println` with a note _there is no action is going to done with the `message`_ is also enough for the first project.